### DISCUSSION TITLE
Ban appeal?

### DISCUSSION THREAD

**Discussion by ghost** [pterodactyl/panel#4210] [General Community]
I can't find a ban appeal place so I'm posting here. A few minutes ago a guy was advertising "free panel and wings setup" in the discord server, and I pinged a few mods with the message "I've been told this is against the rules" to let them know that to the best of my knowledge, this guy is breaking the rules. I immediately get banned and I don't really understand what the point of having moderators is if you can't let the moderators know that someone's doing something that is against the rules, especially if it is someone offering to spoonfeed people. I'm sorry for pinging mods (I pinged 3) but I didn't know what to do in that situation, I promise it won't happen again, can I please be unbanned? 

my ID is 867911574923386940 and my username is Masozi || Tove#9719

I'd also like to mention that I have no prior infractions, warnings, or mutes, so I don't fully understand the insta-ban

![image](https://user-images.githubusercontent.com/74330895/176353227-4498a5aa-8eb3-417e-84e6-21bc5f00efaf.png)

**Comment by iamkubi** [comment]
Unbanned.
